nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -994,6 +1031,257 @@ After going through this checklist, you're ready to go!
 		what.start_piloting(M, "observer") //So theyre unable to fly the pod
 		what.mobs_in_ship += M
 
+/** CHEATSHEET
+
+Fighter bootup sequence components.
+
+Steps:
+Hit ignition switch
+Fuel pump switch
+Engage battery
+Engage APU
+Disengage throttle lock
+Throttle up VERY gently with brakes on so that engine takes over but you're still not moving.
+Disengage APU to let engines take over powergen
+Flight ready.
+
+Shutdown sequence:
+Throttle off + brakes on
+Throttle lock on
+Disengage battery
+Disengage fuel pump (or engine gets flooded)
+Turn off ignition
+
+If you run out of fuel:
+Activate the brakes and begin a shutdown of your fighter. Once you have received more fuel, begin startup sequence as expected. If you run out of fuel, you will be stuck adrift. It is highly recommended that you RTB when you hit 100 fuel as you'll have 30 seconds or so more burn time before you fizzle out.
+
+How to make fuel:
+1 part hydrogen : 1 part carbon to make hydrocarbon. Mix hydrocarbon and welding fuel to produce tyrosene
+
+*/
+
+/obj/structure/overmap/fighter/proc/toggle_canopy()
+	canopy_open = !canopy_open
+	playsound(src, 'nsv13/sound/effects/fighters/canopy.ogg', 100, 1)
+
+/obj/structure/overmap/fighter/verb/show_control_panel()
+	set name = "Show control panel"
+	set category = "Ship"
+	set src = usr.loc
+
+	if(!verb_check())
+		return
+	ui_interact(usr)
+
+/obj/structure/overmap/fighter/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = 0, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state) // Remember to use the appropriate state.
+	if(user != pilot)
+		return
+	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+	if(!ui)
+		ui = new(user, src, ui_key, "fighter_controls", name, 560, 600, master_ui, state)
+		ui.open()
+
+/obj/structure/overmap/fighter/can_move()
+	var/obj/item/twohanded/required/fighter_component/engine/engine = get_part(/obj/item/twohanded/required/fighter_component/engine)
+	if(!engine)

```

Comment:
This should be a switch statement after this.

Reply:
What I was properly meaning is that instead of it being if checks separately this entire block should be a switch if its slightly better with that method as it processes whichever returns true first.